nix/flake-modules/package.nix

```diff
@@ -65,6 +65,9 @@
           mixFodDeps
           ;
 
+        # set the environment variables for the build
+        MIX_ENV = "prod"; # build for production
```

Comment:
Not sure if we should explicitly set it, as we use it in our devenv as well, innit?

Reply:
The devenv environment won't be used during package builds.